traffic_portal/test/end_to_end/conf.js

```diff
@@ -18,14 +18,39 @@
  */
 
 exports.config = {
-    framework: 'jasmine',
-    seleniumAddress: 'http://localhost:4444/wd/hub',
-    specs: ['login_spec.js'],
-    baseUrl: 'https://localhost:8443',
-
-    params: {
-        adminUser: 'admin',
-        adminPassword: 'password'
-   }
-}
+	framework: 'jasmine',
+	seleniumAddress: 'http://localhost:4444/wd/hub',
+	baseUrl: 'https://localhost:4443',
+	getPageTimeout: 30000,
+	// specs: [
```

Comment:
do u want to get rid of this?

Reply:
Comments removed. Individual tests can be ran using protractor conf.js --suite <specname-spec.js>